lib/sr.py

```diff
@@ -41,8 +41,10 @@ def unplug_pbds(self, force=False):
     def all_pbds_attached(self):
         all_attached = True
         for pbd_uuid in self.pbd_uuids():
-            all_attached = all_attached and self.pool.master.xe('pbd-param-get', {'uuid': pbd_uuid,
-                                                                'param-name': 'currently-attached'})
+            all_attached = all_attached and strtobool(self.pool.master.xe('pbd-param-get',
+                                                                          {'uuid': pbd_uuid,
+                                                                           'param-name': 'currently-attached',
+                                                                           }))
```

Comment:
```suggestion
            attached = self.pool.master.xe('pbd-param-get', {'uuid': pbd_uuid, 'param-name': 'currently-attached'})
            all_attached = all_attached and strtobool(attached)
```

might be easier to read